tests/Oryx.Integration.Tests/Python/PythonCustomStartUpCommandTest.cs

```diff
@@ -0,0 +1,124 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Oryx.BuildScriptGenerator.Python;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests
+{
+    [Trait("category", "python")]
+    public class PythonCustomStartUpCommandTest : PythonEndToEndTestsBase
+    {
+        public PythonCustomStartUpCommandTest(ITestOutputHelper output, TestTempDirTestFixture testTempDirTestFixture)
+            : base(output, testTempDirTestFixture)
+        {
+        }
+
+        [Theory]
+        [InlineData("2.7")]
+        [InlineData("3.6")]
+        [InlineData("3.7")]
```

Comment:
add 3.8 as well?

Reply:
Forgot to include 3.8,  I'll add it as well, thanks

Reply:
This got me thinking...the pivot that is being tested is if the user's startup command is being invoked instead of our default command. So for both these test cases here testing with only one version is enough since this decision of whether a user's startup command needs to be used is not dependent on the version.

Reply:
I agree … but I think its better to keep two versions at least .. 2.7 and one of the 3.*

Reply:
Cool, I'll change the versions to include only 2.7 and 3.8.